Lib/test/test_abc.py

```diff
@@ -684,10 +684,16 @@ class B(A, metaclass=abc_ABCMeta, name="test"):
 
     return TestLegacyAPI, TestABC, TestABCWithInitSubclass
 
-TestLegacyAPI_Py, TestABC_Py, TestABCWithInitSubclass_Py = test_factory(abc.ABCMeta,
-                                                                        abc.get_cache_token)
-TestLegacyAPI_C, TestABC_C, TestABCWithInitSubclass_C = test_factory(_py_abc.ABCMeta,
-                                                                     _py_abc.get_cache_token)
+TestLegacyAPI_Py, TestABC_Py, TestABCWithInitSubclass_Py = test_factory(_py_abc.ABCMeta,
+                                                                        _py_abc.get_cache_token)
+TestLegacyAPI_C, TestABC_C, TestABCWithInitSubclass_C = test_factory(abc.ABCMeta,
+                                                                     abc.get_cache_token)
```

Comment:
It looks like the original version constructed the `_Py` variants using `test_factory(abc.ABCMeta, abc.get_cache_token)` and the `_C` variants using `test_factory(_py_abc.ABCMeta, _py_abc.get_cache_token)`. This PR does the inverse. Was the original wrong?

Reply:
Yes, the variables were misnamed. Previously, `TestLegacyAPI_Py` was testing the C implementation and `TestLegacyAPI_C` was testing the Python implementation.